tikv-client/src/main/java/com/pingcap/tikv/TiDBJDBCClient.java

```diff
@@ -68,6 +68,19 @@ public boolean dropTable(String databaseName, String tableName) throws SQLExcept
     }
   }
 
+  // SPLIT TABLE table_name [INDEX index_name] BETWEEN (lower_value) AND (upper_value) REGIONS
+  // region_num
+  public boolean splitTableRegion(
```

Comment:
maybe should firstly check whether current tidb support `split table region`?

Reply:
done.